compiler/cli.py

```diff
@@ -215,6 +215,14 @@ def __init__(self, *args, **kwargs):
             help=" output the preprocessed script",
             action="store_true",
         )
+        self.add_argument(
```

Comment:
This is only useful for the help message, other than that the parsing of the argument just happens outside in pash_init_setup.sh so I am not sure whether all of these options (like nargs, const) are useful